book/src/sec-9-algebra.md

```diff
@@ -61,7 +61,7 @@ Will produce the response:
 ]
 ```
 
-- [ ] [Order By with multiple arguments](https://github.com/naomijub/wooridb/issues/101). The problem here is how to have multiple `.and_then(...)` alter de `partial_cmp`.
+- [ ] [Order By with multiple arguments](https://github.com/naomijub/wooridb/issues/101). The problem here is how to have multiple `.and_then(...)` alter the `partial_cmp`.
```

Comment:
Guess the issue may be wrong as well?

Reply:
Nopes

Reply:
I'm confused here, should I change something?

Reply:
nothings, just it was just zoeira